src/queries/test-id.js

```diff
@@ -11,8 +12,28 @@ function queryAllByTestId(...args) {
 
 const getMultipleError = (c, id) =>
   `Found multiple elements by: [${getTestIdAttribute()}="${id}"]`
-const getMissingError = (c, id) =>
-  `Unable to find an element by: [${getTestIdAttribute()}="${id}"]`
+const getMissingError = (
+  c,
+  id,
+  {computeCloseMatches = false, ...options} = {},
```

Comment:
could be a config instead?

Reply:
Could be both. We have several options that default to global config which you should be able to find around the codebase. Check `waitFor`.